### DISCUSSION TITLE
Getting a ResizeObserver mock to work when testing

### DISCUSSION THREAD

**Discussion by grahamford** [vmware/clarity#5854] [Q&A]
I'm testing a couple of form input fields from the core components on a page. 

The unit tests need a ResizeObserver mock to run in Jest/JSDom. I've mocked this manually in `test-setup.ts` and also by using the npm library `resize-observer-polyfill`. Both methods fix the errors about the ResizeObserver but I then get the message:
```
Uncaught (in promise): TypeError: 'set' on proxy: trap returned falsish for property 'disabled'
    TypeError: 'set' on proxy: trap returned falsish for property 'disabled'
```

Has anyone experienced the same thing, or can give me some advice on what I can do to fix this?

**Comment by mathisscott** [comment]
Can you reproduce it in a stackblitz? It seems like [it may not like how this is getting set here](https://github.com/vmware/clarity/blob/d209ef3082834ba72fdb3b3442504e8784f2358e/packages/core/src/forms/control-group/control-group.element.ts#L171).

A reproduction would help me validate that hunch and be much appreciated.

**Reply by grahamford** [reply]
I couldn't reproduce this in a simple StackBlitz so I created a small Nx application and included features similar to those in my real app (including the Jest configs). However I can't trigger the message about the ResizeObserver. 

I created a Stackblitz here https://stackblitz.com/edit/angular-clarity5-resizeobserver?file=package.json but it won't compile. I'm guessing this is something to do with the Nx configs. There's a copy in the GitHub repo https://github.com/GrahamFord/angular-clarity5-resizeobserver if it helps.

Do you have any suggestions on what I can add to my tests that might trigger the ResizeObserver error?
